### DISCUSSION TITLE
Security Light Support

### DISCUSSION THREAD

**Discussion by Delgrey** [tuya/tuya-home-assistant#237] [Ideas]
Any chance of getting support for these Smart Security lights?

Hardware wise they are good, but the software implementation is limited. I am hoping to get the sensors as an entity in HA, and the lights as another entity, so that I can create automations to say when they come on, at what brightness etc based on motion, and use the motion sensors like any other in HA.

[https://www.aldi.com.au/fileadmin/fm-dam/Products/Special_Buys/Special_Buys_2019/Week_31/SAT/WK31_PD_388x314_46a.jpg](https://www.aldi.com.au/fileadmin/fm-dam/Products/Special_Buys/Special_Buys_2019/Week_31/SAT/WK31_PD_388x314_46a.jpg)

**Comment by iotinkognito** [comment]
Raise as issue rather than discussion,
